plugins/transforms/tableinput/src/main/java/org/apache/hop/pipeline/transforms/tableinput/TableInputDialog.java

```diff
@@ -377,22 +388,27 @@ private List<String> getSqlReservedWords() {
       databaseMetaData = db.getDatabaseMetaData();
       if (databaseMetaData == null) {
         logError("Couldn't get database metadata");
-        return new ArrayList<>();
+        return;
       }
       List<String> sqlKeywords = new ArrayList<>();
       try {
         final ResultSet functionsResultSet = databaseMetaData.getFunctions(null, null, null);
         while (functionsResultSet.next()) {
-          sqlKeywords.add(functionsResultSet.getString("FUNCTION_NAME"));
+          String functionName = functionsResultSet.getString("FUNCTION_NAME");
+          if (functionName.contains(";")) {
+            functionName = functionName.substring(0, functionName.indexOf(";"));
+          }
+          sqlKeywords.add(functionName);
         }
         sqlKeywords.addAll(Arrays.asList(databaseMetaData.getSQLKeywords().split(",")));
       } catch (SQLException e) {
         logError("Couldn't extract keywords from database metadata. Proceding without them.");
       }
-      return sqlKeywords;
+      input.updateKeywords(sqlKeywords);
+      wSql.addLineStyleListener(sqlKeywords);
+      wSql.redraw();
```

Comment:
this line and the previous one look ignored if not executed in the main thread or the open() method